compiler/parser/chapel.lex

```diff
@@ -468,6 +488,50 @@ static const char* eatStringLiteral(yyscan_t scanner, const char* startChar) {
   return astr(stringBuffer);
 }
 
+static const char* eatMultilineStringLiteral(yyscan_t scanner,
+                                             const char* startChar) {
+  YYLTYPE*   yyLloc  = yyget_lloc(scanner);
+  const char startCh = *startChar;
+  int startChCount = 0;
+  int        c       = 0;
```

Comment:
spacing seems really bizarre here

Reply:
Yes it does.  I copied it from eatStringLiteral, but then added startChCount and didn't follow the pattern.  I'll fix up the spacing.